include/zephyr/bluetooth/audio/pacs.h

```diff
@@ -45,6 +45,43 @@ struct bt_pacs_cap {
 	sys_snode_t _node;
 };
 
+/** Structure for registering PACS */
+struct bt_bap_pacs_register_param {
+#if defined(CONFIG_BT_PAC_SNK)
+	/**
+	 * @brief Enables or disables registration of Sink PAC Characteristic.
+	 */
+	bool snk_pac;
+#endif /* CONFIG_BT_PAC_SNK */
+
+#if defined(CONFIG_BT_PAC_SNK_LOC)
+	/**
+	 * @brief Enables or disables registration of Sink Location Characteristic.
+	 *
+	 * Registration of Sink Location is dependent on @ref bt_bap_pacs_register_param.snk_pac
+	 * also being set.
+	 */
+	bool snk_loc;
```

Comment:
Hmm, I'm starting to wonder if we should add `chrc` or `attr` to these names? Or prefix them with `register_`? 

If we ever want to expand `bt_bap_pacs_register_param` to also contain default values, `snk_pac` and `snk_loc` are the obvious names for the default sink PAC record and sink location

Reply:
Don't you think we should make those name changes in such a patch where it becomes relevant then?

Reply:
Possibly. At least while the API is still unstable we can more easily do that, but will be much harder to do later